.github/workflows/pre_release_version_bump.yml

```diff
@@ -29,6 +29,7 @@ jobs:
       - name: Create Pull Request
         uses: peter-evans/create-pull-request@v3
         with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
How about adding in the push trigger temporarily? Once it runs once successfully, we should be good to go. :slightly_smiling_face: 
```
  push:
    branches:
      - version_bump_error
```

Reply:
Good point !

Reply:
```suggestion
          # token: ${{ secrets.GITHUB_TOKEN }}
```